File: Assignment39/Question4.c
#include<stdio.h>
int Minimum(int No)
{
   static int min=0;
   int i=0;
    if(min==0)
    {
        min = No%10;
    }
    else if(No!=0)
    {
        i=No%10;
       printf("%d",i);

        if(min>i)
        {
            min = i;
        }
        No = No/10;
        Minimum(No);
    }
    return min;
}
int main()
{
    int Value=0;
    printf("Enter Number:");
    scanf("%d",&Value);

    int iret = Minimum(Value);
    printf("Minimum digit of Number is:%d\n",iret);
    return 0;
}

File: Assignment7/Question4.java
import java.util.*; 
class Check
{
    int CountOdd(int iNo)
    {
        int iSum=1;
        while (iNo>0) {
            int iDigit = iNo%10;   
            {
                iSum=iSum*iDigit;
            }
            iNo = iNo/10;
        }       
    return iSum;
    }
}
class Question4{
    public static void main(String args[]) {
    
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter Number:");
        int iValue = sobj.nextInt();
        Check cobj = new Check();
        int iRet = cobj.CountOdd(iValue);
        System.out.printf("%d is Multiplication of Digit.\n",iRet);
    }
}